opendevin/server/listen.py

```diff
@@ -272,12 +272,7 @@ async def websocket_endpoint(websocket: WebSocket):
     session = session_manager.add_or_restart_session(sid, websocket)
     await websocket.send_json({'token': token, 'status': 'ok'})
 
-    latest_event_id = -1
-    if websocket.query_params.get('latest_event_id'):
-        latest_event_id = int(websocket.query_params.get('latest_event_id'))
```

Comment:
I think this is actually used? Can we double check if the `latest_event_id` is used in the FE?

https://github.com/OpenDevin/OpenDevin/blob/a2ea17909d1d7081d2ce677a7b94a5075e0282cb/frontend/src/services/session.ts#L63-L65

Reply:
Within that standalone method, it is just a local variable without any outside reference.

Reply:
If that's the case, should we delete this from the FE too?

Reply:
Honestly, not sure what the initial intention was here.
As far as I tracked it down, it came in with #1810 in May by @rbren and it hadn't changed since then.
So either this was intended to be passed on here, but idk how.
What do you think?

Reply:
If I see this right, this bit is trying to limit the events sent to the FE to those after `latest_event_id`. As far as backend is concerned, that value doesn't actually limit the session, it won't give events earlier than your session anyway. But when the session is loaded, it might be that we wanted to limit the messages sent to FE for display to the user? I had sessions with over 2k events in the stream, if we send all those to FE the poor chat box might not be too happy.

Just guessing here, I know this bit of code is in BE for a long time, but I don't remember seeing it actually limit any session yet. If this guess is remotely correct, though, we might want to implement something in FE for that?

Reply:
I can only _remove_ the code from the FE 😁 
If we want the FE to actually do something with that param, I'd rather have a specialist do that.